formule_card.php

```diff
@@ -518,6 +497,44 @@
 	include DOL_DOCUMENT_ROOT.'/core/tpl/card_presend.tpl.php';
 }
 
+if (!empty($conf->use_javascript_ajax)) { ?>
+    <script type="text/javascript">
+    $(document).ready(function () {
+        $("#fk_country").change(function() {
+            $("#fk_country").trigger("change.select2"); // Déclencher manuellement la mise à jour de Select2
```

Comment:
il me semble que tu as pas besoin de faire ça et tu peux directement utiliser $(this).val()